src/AWS/Orleans.Streaming.SQS/Streams/SQSBatchContainer.cs

```diff
@@ -8,17 +7,20 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Text;
+using Amazon.SQS;
+using Amazon.SQS.Model;
 using SQSMessage = Amazon.SQS.Model.Message;
 
 namespace OrleansAWSUtils.Streams
 {
     [Serializable]
     [Orleans.GenerateSerializer]
-    internal class SQSBatchContainer : IBatchContainer
+    public class SQSBatchContainer : IBatchContainer
     {
         [JsonProperty]
         [Orleans.Id(0)]
-        private EventSequenceTokenV2 sequenceToken;
+        private StreamSequenceToken sequenceToken;
```

Comment:
I think it's fine but we ahould really double check that it doesn't break backward compat